src/celeritas/phys/Primary.hh

```diff
@@ -27,7 +27,7 @@ struct Primary
     Real3 direction{0, 0, 0};
     real_type time{};
     EventId event_id;
+    real_type weight{1.0};
 };
-
```

Comment:
We tend to leave some whitespace in between blocks of code for style and vim navigation (left and right braces move between blank lines) so please revert this whitespace change and the one in Secondary.